service/forward_service.hh

```diff
@@ -131,6 +131,7 @@ class forward_service : public seastar::peering_sharded_service<forward_service>
     } _stats;
     seastar::metrics::metric_groups _metrics;
 
+    optimized_optional<abort_source::subscription> _early_abort_subscription;
```

Comment:
Why does this need to be an optional? As far as I can tell (maybe I'm missing something), this class has just one constructor, which always sets _early_abort_subscription.